manticore/core/cpu/disasm.py

```diff
@@ -141,6 +138,18 @@ def unimplemented(self, il):
         return (il.operation == enums.LowLevelILOperation.LLIL_UNIMPL or
                 il.operation == enums.LowLevelILOperation.LLIL_UNIMPL_MEM)
 
+    def _get_current_llil_func(self, code, pc):
+        if pc in self.llil_func_cache:
+            return self.llil_func_cache[pc]
+
+        from binaryninja import Architecture, LowLevelILFunction
```

Comment:
We want to keep this internal to the function since a user might not have binaryninja installed, thus we don't want to do get import errors. You mean in the top of the function? If it is cached won't we be doing an import that is not used?